gemfiles/rails_4_2.gemfile

```diff
@@ -9,6 +9,5 @@ gem "pry"
 gem "railties", "~> 4.2.0"
 gem "actionpack", "~> 4.2.0"
 gem "activemodel", "~> 4.2.0"
-gem "nokogiri", "< 1.7" if RUBY_VERSION < "2.1"
```

Comment:
... I didn't remove it? It might have gotten nixed when I ran `appraisal install`.